requirements-docs.txt

```diff
@@ -1,27 +1,27 @@
 # You should not edit this file directly.  Instead, you should edit requirements-docs.txt.in.
 alabaster==0.7.12
-Babel==2.7.0
+Babel==2.8.0
 certifi==2019.11.28
 chardet==3.0.4
-docutils==0.15.2
+docutils==0.16
```

Comment:
docutils are different here and within requirements.txt just to make you aware.

Reply:
Good catch. I anticipate both requirements files will be independent and it would not be typical to install them both in the same environment, but better safe than sorry

Reply:
fixed in 4096e1d